doc/config.md

```diff
@@ -65,6 +65,7 @@ system:
     teamcity: true
   debug: true
   parallelLimit: 3
+  parallelWorkersLimit: 4
```

Comment:
rename it to `workers`, please. It should be consistent with hermione `workers` option